feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
It's really good that you have decided to add this line of code

Reply:
Thanks for this comment, Bogdan!

Reply:
@bstashchuk I'm in the Brave Browser and I am able to send you a tip, but it says that you've not yet verified.  Have you considered doing that? It's a way for us to tip the content creators whom we love.  Thank you for this wonderful course!  I really love it so far!

Reply:
![image](https://user-images.githubusercontent.com/9057522/106185820-faf84600-6168-11eb-9517-08f6b9d2cb6e.png)
See the Tip button?

Reply:
Hi Bogdan, liking the course.. Just testing out the comments on the pull requests section of the course.

Reply:
I'm sure it's really good that you have decided to add this line code, **just very good**

Reply:
I do, indeed, think that adding this line of code has been very important in the grand scheme of things.

Reply:
Great course!

Reply:
Thank a lot Bogdan! Enjoying this course. Best about Git that I've seen so far)

Reply:
This is a great course, bogdan. Really enjoying the detail concepts of Git and Github. You explain it well by focusing on each topic and explaining step by step.